### PR TITLE
Add instruction to configure remote recipe repo in setup

### PR DESCRIPTION
**Pull request by AdrianDAlessandro**
Decision has been made to ensure everyone is on the same page at the start of the course by having the recipe repo configured with a remote before starting.

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -2,19 +2,42 @@
...
 ---
 
-Completing this course requires you to have access to a computer with Git
-installed. This course is currently being delivered remotely so please make sure
-you have access to a suitable computer. All attendees should download the zip
-file and, if not already available, install Git.
+Completing this course requires you to have access to a computer with Git installed and
+a [GitHub](https://github.com) account. This course is currently being delivered in
+person and remotely so  please make sure you have access to a suitable computer. All
+attendees should configure their recipe repository and be comfortable with using Git.
```
Comment (dc2917) [inline setup.md:8]:
```suggestion
person and remotely so please make sure you have access to a suitable computer. All
attendees should configure their `recipe` repository and be comfortable with using Git.
```

File: setup.md [hunk 2]
```diff
@@ -2,19 +2,42 @@
...
 
-## Downloading the Zip File
+## Configure your recipe repository
+
+If you **have** completed our
+[Introduction to using Git and GitHub for software development][intro-course] course,
+you should have a `recipe` repository in your GitHub account that will be used in this
```
Comment (dc2917) [inline setup.md:14]:
```suggestion
you should have a local `recipe` repository, and an associated remote repository in your GitHub account that will be used in this
```

File: setup.md [hunk 3]
```diff
@@ -2,19 +2,42 @@
...
 
-The zip archive containing files that will be used in the session is available
-[here](code/recipe_with_history.zip). Please download and **save it in your home directory,
-do not extract it yet**.
 
 ## Install Git
 
+> ## Important!
+>
+> If you do not already have Git installed and are comfortable with using it for simple
```
Comment (dc2917) [inline setup.md:33]:
```suggestion
> If you do not already have Git installed or are not comfortable with using it for simple
```

**Review by dc2917 — CHANGES_REQUESTED** [review]
Looks good to me, just a couple of typo changes, which might also affect line lengths

**Review by dc2917 — APPROVED** [review]

File: setup.md [hunk 4]
```diff
@@ -2,19 +2,42 @@
...
+If you **have not** completed the intro course:
+- Download [this zip archive](code/recipe_with_history.zip) that contains the `recipe`
+repo with some previous commit history that will be used in the session.
+- Extract it into your home directory.
+- Create a repository on your personal GitHub account and set it as the remote for this
+`recipe` repository you have locally.
+
+If you do not know how to do this, you can follow the steps in the "Configuring
+a remote repository from a local one" exercise in
+[this lesson.](https://imperialcollegelondon.github.io/introductory_grad_school_git_course/l2-02-remote_repositories/index.html)
```
Comment (dalonsoa) [inline setup.md:26]: If you do not know how to do this... why are you in the intermediate course? 

Seriously, it is good to describe this in detail, but my joke has a point, I think...

**Review by dalonsoa — APPROVED** [review]
All looks good to me!

File: setup.md (hunk 4 above)
Comment (dalonsoa) [reply setup.md:26]: Fair point

Comment (alexdewar) [reply setup.md:26]: If you only ever create your git repos by doing a "git clone" it's possible to never have to manually add a remote, too!

**Review by alexdewar — APPROVED** [review]

**Pull request merged**